### PR TITLE
Updated backend tutorial

### PR DESCRIPTION
**Pull request by antonpirker**
Brought the Backend tutorial up to date to the new SDK api.

I also updated the demo repo that goes along with the turial: https://github.com/sentry-tutorials/backend-monitoring

(But I have no permissions yet to push to that repo, so the branch is only on my machine, will update with a link to the PR as soon as I have the permissions)

## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [X] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by lizokm — APPROVED** [review]
Thank you so much for taking a look at this and making updates! I've added a few more content updates. Excited to get this refreshed! And yes, we should definitely add a link to this in the Django SDK.

File: docs/product/sentry-basics/integrate-backend/getting-started.mdx [hunk 1]
```diff
@@ -56,78 +55,67 @@ Sentry captures data by using a platform-specific SDK within your application ru
...
 To build and run the demo application on your localhost:
 
 1. Open a shell terminal and change directory to the `backend-monitoring` project root folder.
 
-2. If you haven't installed Python3, do so by running the following:
+2. If you haven't installed Python, do so by running the following:
 
    ```bash
-    brew install python3
+   brew install python
```
[the remark this answers is not in the document]
Comment (antonpirker) [reply docs/product/sentry-basics/integrate-backend/getting-started.mdx:105]: Its a good idea, but on other system it is more than one line and there are tons of ways to install on linux. We could link to this (or something similar) https://python.land/installing-python

**Comment by antonpirker** [comment]
I guess we can just merge this now and for the improvements to "how to install python" make a separate pr.

**Comment by antonpirker** [comment]
FYI: updated the sample repo too: https://github.com/sentry-tutorials/backend-monitoring @lizokm

**Pull request merged**